pkg/importer/dataStream_test.go

```diff
@@ -288,16 +288,15 @@ func TestCopyImage(t *testing.T) {
 
 func createTestData() map[string]string {
 	imageDir, _ := filepath.Abs(testImagesDir)
-
-	// xz data
+	
 	xzfile, _ := framework.FormatTestData(filepath.Join(imageDir, "tinyCore.iso"), os.TempDir(), image.ExtXz)
 	gzfile, _ := framework.FormatTestData(filepath.Join(imageDir, "tinyCore.iso"), os.TempDir(), image.ExtGz)
-	xztarfile, _ := framework.FormatTestData(filepath.Join(imageDir, "tinyCore.iso"), os.TempDir(), []string{image.ExtTar, image.ExtGz}...)
+	tarfile, _ := framework.FormatTestData(filepath.Join(imageDir, "tinyCore.iso"), os.TempDir(), image.ExtTar)
 
 	return map[string]string{
 		".xz":     xzfile,
 		".gz":     gzfile,
-		".tar.xz": xztarfile,
+		".tar":    tarfile,
```

Comment:
should we test other combos, eg _.tar.xz_, _.tar.gz_, _.gz_, _.xz_, and _.qcow2_ variants?